RangeHighlight/i18n/fr.json

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022-2024 Jamie Taylor
+// Translation by Tenebrosful
+{
+  // Strings used in the config menu
+  "config.install-gmcm-options": "Si vous avez installé le mod GMCM Options, vous serez en capacité de configurer les couleurs dans ce menu.",
+  "config.refresh-interval": "Interval de rafraîchissement",
+  "config.refresh-interval.tooltip": "À quelle fréquence (\"ticks\" en jeu) la position de la tuile mise en avant est calculée",
+  "config.enable": "Activer",
+  "config.hotkey": "Touche de raccourcis",
+  "config.tint": "Couleur",
+  "config.show-others": "Afficher les autres",
+  "config.hotkeys-toggle": "Maintenir la touche",
+  "config.hotkeys-toggle.tooltip": "Si cochée, les touches de raccourcis basculent l'activation de l'affichage de la portée; sinon la portée ne s'affiche que si la touche est maintenue.",
+  "config.show-all-key": "Afficher toutes les portées",
+  "config.show-all-key.tooltip": "Touche de raccourcis pour afficher toutes les portées",
+  "config.highlight-building-on-mouseover": "Portée des batiments au survol",
+  "config.highlight-building-on-mouseover.tooltip": "If true then highlighters for buildings (e.g., Junimo Huts) run when the mouse is pointing at the building.",
```

Comment:
Looks like you missed a line

Reply:
Yes and also changing `config.hotkeys-toggle` because I literally translate to the opposite and figure out when playing